### DISCUSSION TITLE
芙芙天下第一可爱！

### DISCUSSION THREAD

**Discussion by ywzPan** [share121/inter-knot#2077] [其他]
![ae327330-e4bb-4693-b4c9-e1da6822adb7](https://github.com/user-attachments/assets/c694ef1e-3931-4775-80b2-5cc7a2c1e526)
**我宣布，芙芙天下第一可爱！**

**Comment by shydrogen** [comment]
头像
![QQ图片20231111182956](https://github.com/user-attachments/assets/a757832a-56e0-4634-900d-4c1ea59233f2)
